oddbird/accoutrement
# Modules type

![](https://source.unsplash.com/featured/?cute,animal&tked)




## Description
Upgrading the `type` module  with Sass module usage. 

## Steps to test/reproduce

See files in `./sass/type/`

_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._


## Show me
_Provide screenshots/animated gifs/videos if necessary._
